### PR TITLE
fix: use single pvc

### PR DESCRIPTION
**Pull request by hummingalpaca**
Reduce the number of PVCs required per container to stay below azure vm disk limits

### CODE REVIEW

File: .internal-ci/charts/reserve-auditor/templates/statefulset.yaml [hunk 1]
```diff
@@ -72,9 +64,9 @@ spec:
             failureThreshold: 4
             periodSeconds: 30
           volumeMounts:
-            - name: {{ $.Chart.Name }}-ledger
+            - name: {{ $.Chart.Name }}
```
Comment (jgreat) [inline .internal-ci/charts/reserve-auditor/templates/statefulset.yaml:67]: I don't think you're going to be able to reuse the same volume `{{ $.Chart.Name }}` here like this.  You probably need to rewire the container/app to put `/ledger` and `/mobilecoind` under a single path like `/data`  

So you end up with a single volume  mount at `/data`  and sub directories created by the app/entrypoint for `/data/ledger`, `/data/mobilecoind` ...

**Review by jgreat — CHANGES_REQUESTED** [review]
See the inline comment on the volumes.  Unless I'm missing something we may need to do some work on the app config level to consolidate the storage.

File: .internal-ci/charts/reserve-auditor/templates/statefulset.yaml (hunk 1 above)
Comment (jgreat) [reply .internal-ci/charts/reserve-auditor/templates/statefulset.yaml:67]: There are a few other places that might need the same sub directory patterns.

Comment (jgreat) [reply .internal-ci/charts/reserve-auditor/templates/statefulset.yaml:67]: Well I stand corrected,  I guess that will work within a pod.

**Review by jgreat — APPROVED** [review]

**Review by david1alvarez — APPROVED** [review]
looks good, linting failure seems likely due to an outdated `main` CI.yml file. Feel free to merge, but lets try to nail down whats causing that issue if it persists in future PRs

**Pull request merged**